src/plugins/doris/RecordingRules/Queries/index.tsx

```
@@ -0,0 +1,61 @@
+import DocumentDrawer from '@/components/DocumentDrawer';
+import GraphPreview from '../../AlertRule/GraphPreview';
+
+interface IProps {
+  datasourceValue: number;
+  field: any;
+  prefixPath: (string | number)[];
+  path: (string | number)[];
+}
+
+export default function index(props: IProps) {
+  const { t, i18n } = useTranslation(NAME_SPACE);
```

Biome (2.4.4)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)